apps/cevent_frontend/src/app/my_tickets/page.tsx

```diff
@@ -1,7 +1,36 @@
+'use client'
+import { redirect, useRouter } from 'next/navigation';
+import { routes } from '@/utils/navigation/Routes';
+import { Button } from "@mui/material";
+import { useSession } from "next-auth/react";
+import { useEffect, useState } from "react";
+
 export default function MyTicketsPage() {
```

Comment:
Why does my ticket page currently require me to check the authentication status?